std/range/package.d

```diff
@@ -7149,6 +7210,43 @@ if (isForwardRange!Source)
 
```

Comment:
I would add the following example to show the new functionality

```d
int i;
auto inputRange = generate!(() => ++i).take(10);
auto chunked = inputRange.chunks(2);

assert(inputRange.front.equal([1, 2]));
assert(inputRange.front.empty);
inputRange.popFront;
assert(inputRange.front.equal([3, 4]));
```

or something similar